### ISSUE TITLE
Icons hide in by css in last update

### ISSUE THREAD

**Issue by strtob** [kartik-v/yii2-tabs-x#75]
Hi,

Icons in labels will be hide (output as example):

`<a class="nav-link active" href="#w33-tab0" data-toggle="tab" role="tab" aria-selected="false" aria-controls="w33-tab0"><i class="fas fa-home"></i> Basic</a>`

this css should be responsible for:

```
.tabs-left > .nav-tabs .nav-item .nav-link :before, .tabs-right > .nav-tabs .nav-item .nav-link :before {
    position: absolute;
    height: 100%;
    content: " ";
    width: 1px;
    border-color: rgba(0,0,0,.15) #fff;
}
```
Are there any ideas how to disable/overwrite?

Thanks a lot,
Tobi

**Comment by kartik-v** [comment]
Fixed via kartik-v/bootstrap-tabs-x#36

**Comment by kartik-v** [comment]
Fixed via this [bootstrap tabs-x enhancement](https://github.com/kartik-v/bootstrap-tabs-x/commit/12e5eeceda5ecbdf1bc7e429c4cea69c0e0b5267)
